plasmapy/utils/decorators/converter.py

```diff
@@ -11,6 +11,28 @@
 
 
 def angular_freq_to_hz(fn):
+    """
+    Converts angular frequency (ω) frenquency to Hertz(hz)
```

Comment:
This needs to be more descriptive of what the decorator is doing.  Something along the lines off...

"A decorator that adds to a function the ability to convert the function's return from angular frequency (rad/s) to frequency (Hz).

A `kwarg` `to_hz` is added to the function's signature, with a default value of `False`.  The keyword is also added to the function's docstring under the **Other Parameters** heading."